tutorials/fit/NumericalMinimization.py

```diff
@@ -0,0 +1,85 @@
+#  \file
+#  \ingroup tutorial_fit
+#  \notebook -nodraw
+#  Example on how to use the new Minimizer class in ROOT
+#   Show usage with all the possible minimizers.
+#  Minimize the Rosenbrock function (a 2D -function)
+#  This example is described also in
+#  http://root.cern.ch/drupal/content/numerical-minimization#multidim_minim
+#  input : minimizer name + algorithm name
+#  randomSeed: = <0 : fixed value: 0 random with seed 0; >0 random with given seed
+#
+#  \macro_code
+#
+#  \author Lorenzo Moneta
+
+import ROOT
+
+def RosenBrock( vecx ):
+   x = vecx[0]
+   y = vecx[1]
+   return (y-x**2)**2 + (1-x)**2
+
+# create minimizer giving a name and a name (optionally) for the specific algorithm
+#  possible choices are:
+#      minimizerName                  algoName
+#     Minuit                     Migrad, Simplex,Combined,Scan  (default is Migrad)
+#      Minuit2                    Migrad, BFGS, Simplex,Combined,Scan  (default is Migrad)
+#     GSLMultiMin                ConjugateFR, ConjugatePR, BFGS, BFGS2, SteepestDescent
+#     GSLSimAn
+#     Genetic
+def NumericalMinimization(minimizerName = "Minuit2",
+                          algoName = "" ,
+                          randomSeed = -1) :
+
+   minimizer = ROOT.Math.Factory.CreateMinimizer(minimizerName, algoName);
+   if (not minimizer) :
+      print("Error: cannot create minimizer \"",minName,"\". Maybe the required library was not built?")
```

Comment:
There are a few problems here. The first is that the `minName` variable is not defined. I guess you mean `minimizerName`. Second, I don't think this way of using print is what we want:
Python 2 will print it as a tuple
```python
>>> a = "here"
>>> print("This is it: ", a, " --> hello")
('This is it: ', 'here', ' --> hello')
```
Python 3 will add unnecessary whitespace to the string.
```
>>> a = "here"
>>> print("This is it: ", a, " --> hello")
This is it:  here  --> hello
```

The more correct way would be using `format`, which is Python2 and 3 compatible:
```python
print("Error: cannot create minimizer \"{}\". Maybe the required library was not built?".format(minimizerName))
```